environments/values-idfint.yaml

```diff
@@ -28,7 +28,7 @@ linters:
 mobu:
   enabled: true
 moneypenny:
-  enabled: true
+  enabled: false
```

Comment:
I think we still need this, though, right?

Reply:
Whoops.  Yes.